pkg/plugins/common/kustomize/v1/scaffolds/init.go

```diff
@@ -80,5 +80,7 @@ func (s *initScaffolder) Scaffold() error {
 		&kdefault.ManagerConfigPatch{},
```

Comment:
IMO it would be more an RFE and not issue.
Thereby the title would be like:

> `:warning: Add RBAC permissions to allow Prometheus Operator have access to the metrics`

Reply:
changed